shared/utils/verified-content.ts

```diff
@@ -24,3 +24,29 @@ export enum SPCPFieldTitle {
 export enum SgidFieldTitle {
   SgidNric = 'sgID Validated NRIC',
 }
+
+export const VerifiedKeyToSPCPTitleMap: Record<VerifiedKeys, SPCPFieldTitle> = {
+  [VerifiedKeys.SpUinFin]: SPCPFieldTitle.SpNric,
+  [VerifiedKeys.CpUen]: SPCPFieldTitle.CpUen,
+  [VerifiedKeys.CpUid]: SPCPFieldTitle.CpUid,
+  [VerifiedKeys.SgidUinFin]: SPCPFieldTitle.SpNric, // safeguarding for backwards compatibility
+}
+
+/**
+ * Maps SPCP VerifiedKeys to their titles. Used during decryption/population of outputs.
```

Comment:
nit: grammatical error in the docs? `handle and retain in -> it during mapping`?

suggestion: could we provide an @example annotation? 
eg, given cpUid Step-0 -> produces Corpass-step-0